### PR TITLE
Feature/drawer navigation menu

### PR DESCRIPTION
**Pull request by Adaok**
Add the Drawer Navigation Menu.

### CODE REVIEW

File: ComAutis/app/src/main/java/com/lpiem_lyon1/comautis/Adapters/DrawerAdapter.java [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+        TextView textViewItemName = (TextView) layout.findViewById(R.id.tv_drawer_item);
+        ImageView iconViewTtem = (ImageView) layout.findViewById(R.id.icon_drawer_item);
+
+        textViewItemName.setText(mDrawerItemList.get(position).getmName());
+        iconViewTtem.setImageResource(mDrawerItemList.get(position).getmIconId());
+
+        return layout;
+    }
+
+    public List<DrawerItem> getmDrawerItemList() {
```
Comment (Alexisdum) [inline ComAutis/app/src/main/java/com/lpiem_lyon1/comautis/Adapters/DrawerAdapter.java]: Delete

File: ComAutis/app/src/main/java/com/lpiem_lyon1/comautis/Adapters/DrawerAdapter.java [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+        return mDrawerItemList;
+    }
+
+    public void setmDrawerItemList(List<DrawerItem> mDrawerItemList) {
```
Comment (Alexisdum) [inline ComAutis/app/src/main/java/com/lpiem_lyon1/comautis/Adapters/DrawerAdapter.java]: Delete

File: ComAutis/app/src/main/java/com/lpiem_lyon1/comautis/BaseActivity.java [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+
+        //init drawer menu item
+        mDrawerItemList = new ArrayList<>();
+        mDrawerItemList.add(new DrawerItem(getString(R.string.child_list),R.drawable.ic_child_face));
+        mDrawerItemList.add(new DrawerItem(getString(R.string.library),R.drawable.ic_library));
+
+        //init list view with menu items
+        DrawerAdapter drawerAdapter = new DrawerAdapter(mContext , mDrawerItemList);
+        mDrawerListView.setAdapter(drawerAdapter);
+
```
Comment (Alexisdum) [inline ComAutis/app/src/main/java/com/lpiem_lyon1/comautis/BaseActivity.java]: Delete some space

File: ComAutis/app/src/main/res/layout/drawer_item_row.xml [hunk 4]
```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout
+    xmlns:android="http://schemas.android.com/apk/res/android"
+    android:orientation="horizontal"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent">
+
+    <ImageView
+        android:layout_width="50dp"
```
Comment (ghost) [inline ComAutis/app/src/main/res/layout/drawer_item_row.xml]: Mettre les taille dans des dimens

File: ComAutis/app/src/main/res/values-fr/strings.xml [hunk 5]
```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <string name="app_name">ComAutis</string>
+    <string name="child_list">Liste des enfants</string>
```
Comment (ghost) [inline ComAutis/app/src/main/res/values-fr/strings.xml:7]: Ranges tes strings

**Pull request merged**